js-extensions/package.json

```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "js-yaml": "^3.6.0",
+    "envify": "3.4.1",
```

Comment:
@michaelneale yeah, is needed to apply the process env settings, which are used in the react code to enable/disable dev mode.

Boring detail (ignore at will) .... it's a browserify transform that looks in the code for `process.env.NODE_ENV` and simply replaces them with strings so that it works in the browser.